### ISSUE TITLE
Crash when player opens inventory

### ISSUE THREAD

**Issue by Lolothepro** [Sirttas/ElementalCraft#157]
[crash-2024-02-25_20.40.29-client.txt](https://github.com/Sirttas/ElementalCraft/files/14397579/crash-2024-02-25_20.40.29-client.txt)

```
java.lang.NullPointerException: Trying to access unbound value: ResourceKey[minecraft:item / patchouli:guide_book]
	at net.neoforged.neoforge.registries.DeferredHolder.value(DeferredHolder.java:102) ~[neoforge-20.4.181-universal.jar%23173!/:?] {re:mixin,re:classloading}
	at net.neoforged.neoforge.registries.DeferredHolder.get(DeferredHolder.java:116) ~[neoforge-20.4.181-universal.jar%23173!/:?] {re:mixin,re:classloading}
	at sirttas.elementalcraft.item.ECCreativeModeTabs.createElementopedia(ECCreativeModeTabs.java:262) ~[elementalcraft-1.20.4-6.1.1.jar%23170!/:1.20.4-6.1.1] {re:classloading}
	at sirttas.elementalcraft.item.ECCreativeModeTabs.generateElementopedia(ECCreativeModeTabs.java:257) ~[elementalcraft-1.20.4-6.1.1.jar%23170!/:1.20.4-6.1.1] {re:classloading}
	at sirttas.elementalcraft.item.ECCreativeModeTabs.lambda$static$1(ECCreativeModeTabs.java:168) ~[elementalcraft-1.20.4-6.1.1.jar%23170!/:1.20.4-6.1.1] {re:classloading}
	at net.neoforged.neoforge.event.EventHooks.onCreativeModeTabBuildContents(EventHooks.java:934) ~[neoforge-20.4.181-universal.jar%23173!/:?] {re:classloading}
	at net.minecraft.world.item.CreativeModeTab.buildContents(CreativeModeTab.java:150) ~[client-1.20.4-20231207.154220-srg.jar%23172!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.world.item.CreativeModeTabs.lambda$buildAllTabContents$50(CreativeModeTabs.java:1888) ~[client-1.20.4-20231207.154220-srg.jar%23172!/:?] {re:classloading,pl:accesstransformer:B}
	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184) ~[?:?] {}
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179) ~[?:?] {}
	at java.util.Iterator.forEachRemaining(Iterator.java:133) ~[?:?] {re:mixin}
	at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1939) ~[?:?] {}
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509) ~[?:?] {}
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499) ~[?:?] {}
	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151) ~[?:?] {}
	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174) ~[?:?] {}
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?] {}
	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596) ~[?:?] {}
	at net.minecraft.world.item.CreativeModeTabs.buildAllTabContents(CreativeModeTabs.java:1888) ~[client-1.20.4-20231207.154220-srg.jar%23172!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.world.item.CreativeModeTabs.tryRebuildTabContents(CreativeModeTabs.java:1897) ~[client-1.20.4-20231207.154220-srg.jar%23172!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.client.gui.screens.inventory.CreativeModeInventoryScreen.<init>(CreativeModeInventoryScreen.java:122) ~[client-1.20.4-20231207.154220-srg.jar%23172!/:?] {re:classloading,pl:runtimedistcleaner:A}
```

**Comment by Sirttas** [comment]
This is an oversight, I forgot to check if patchouli is resent or not.

**Comment by Sirttas** [comment]
Fixed in 6.1.2
